openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -64,6 +65,10 @@ const TagsPage = () => {
   const [editTag, setEditTag] = useState<TagClass>();
   const [error, setError] = useState<string>('');
   const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [errorDataCategory, setErrorDataCategory] =
+    useState<{ [key: string]: string | undefined }>();
+  const [errorDataTag, setErrorDataTag] =
+    useState<{ [key: string]: string | undefined }>();
```

Comment:
instead of repeating `{ [key: string]: string | undefined }` we can make a separate type and use it here.
```type keyValue = { [key: string]: string | undefined };```